Composer/packages/client/src/components/Modal/dialogStyle.ts

```diff
@@ -1,10 +1,30 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+import { css } from '@emotion/core';
+
 /**
  * Well known style names for the AlertDialog and ConfirmDialog.
  */
 export const dialogStyle = {
```

Comment:
Constants used across dialogs components and shared styles are two different things.  I recommend a constants.ts and putting styles within the components.